### PR TITLE
chore(connectors): update docs for rabbitMQ, split routing data

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## What is the purpose of the change

updated docs for rabbitMQ, reason splited routing data in template :  https://github.com/camunda/web-modeler/pull/3613

## Are there related marketing activities

https://github.com/camunda/team-connectors/issues/317

## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/connectors/out-of-the-box-connectors/rabbitmq.md [hunk 1]
```diff
@@ -64,36 +64,16 @@ To connect with credentials, take the following steps:
...
 For a **URI** type connection, the required fields are `exchange` and `routingKey`.
 For a **Credentials** type connection, the required fields are `exchange`, `routingKey`, `virtualHost`, `hostName`, and `port`.
 
-![connectors-rabbitmq-routing](../img/connectors-rabbitmq-routing.png)
+- Go to [RabbitMQ Exchanges, routing keys and bindings documentation](https://www.cloudamqp.com/blog/part4-rabbitmq-for-beginners-exchanges-routing-keys-bindings.html) for learn more about RabbitMQ exchanges and routing keys.
+- Go to [RabbitMQ virtual host documentation](https://www.rabbitmq.com/vhosts.html) for learn more about RabbitMQ virtual hosts.
+- Go to [RabbitMQ networking documentation](https://www.rabbitmq.com/networking.html) for learn more about configuration RabbitMQ host and port.
+
+![connectors-rabbitmq-routing](../img/connectors-rabbitmq-routing-uri.png)
+![connectors-rabbitmq-routing](../img/connectors-rabbitmq-routing-cred.png)
```
Comment (chillleader) [inline docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:76]: A more concise language here might be more readable. What do you think about this?

```suggestion
Please refer to RabbitMQ documentation to learn about routing attributes:
- [Exchanges, routing keys and bindings](https://www.cloudamqp.com/blog/part4-rabbitmq-for-beginners-exchanges-routing-keys-bindings.html)
- [Virtual hosts](https://www.rabbitmq.com/vhosts.html)
- [Networking, host and port configuration](https://www.rabbitmq.com/networking.html)
```

Comment (christinaausley) [reply docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:76]: I really like this suggestion! I would just adjust to:

~~~Please~~~ Refer to RabbitMQ documentation to learn about routing attributes:
- [Exchanges, routing keys**,** and bindings](https://www.cloudamqp.com/blog/part4-rabbitmq-for-beginners-exchanges-routing-keys-bindings.html)
- [Virtual hosts](https://www.rabbitmq.com/vhosts.html)
- [Networking, host**,** and port configuration](https://www.rabbitmq.com/networking.html)

Comment (christinaausley) [reply docs/components/connectors/out-of-the-box-connectors/rabbitmq.md:76]: Adjusting this in my technical review/latest commit. 👍

**Review by christinaausley — APPROVED** [review]
Technical review in my latest commit, which adjusted some wording for clarity, incorporated the comment from @chillleader, and also moved the screenshots around to the referenced content. Let me know if you have any questions about these changes.

**Review by igpetrov — APPROVED** [review]
Thank you

**Pull request merged**